### DISCUSSION TITLE
"controller-manager" secret is deleted when helm upgrade is executed for actions runner controller

### DISCUSSION THREAD

**Discussion by navachaitanyak** [actions/actions-runner-controller#2693] [Questions]
Hello 
I'm deploying the action runner controller in the Kubernetes cluster and whenever I run the helm chart upgrade using the below command, during deployment "controller-manager" is deleted, how to avoid the deletion of "controller-manager" secret 

helm upgrade command
`helm upgrade --install --namespace arc-dev --wait actions-runner-controller ./actions-runner-controller --values ./actions-runner-controller/values.yaml`

Below is the log for reference:
`client.go:684: [debug] Patch Deployment "actions-runner-controller" in namespace arc-dev
client.go:417: [debug] Created a new MutatingWebhookConfiguration called "actions-runner-controller-mutating-webhook-configuration" in 
client.go:417: [debug] Created a new ValidatingWebhookConfiguration called "actions-runner-controller-validating-webhook-configuration" in 
client.go:445: [debug] Deleting Secret "controller-manager" in namespace arc-dev...
upgrade.go:396: [debug] waiting for release actions-runner-controller resources (created: 2 updated: 9  deleted: 1)
wait.go:48: [debug] beginning wait for 11 resources with timeout of 5m0s`

**Comment by navachaitanyak** [comment]
@mumoshu can you please help me on this?

**Reply by ghost** [reply]
I am facing the same issue, can someone help?
